### ISSUE TITLE
path is not correct in the options page

### ISSUE THREAD

**Issue by ErikKrause** [andy-portmen/auto-shutdown#6]
... but there is no path to edit on options page. Also it is not clear which application can not be accessed, since it says "Native client is installed!"

Windows 10, both FF 56 and 58

**Comment by andy-portmen** [comment]
What is the command that you use in the options page? Can you take a screenshot and share here?

**Comment by ErikKrause** [comment]
I use the Shutdown command as preset for Windows: shutdown /s /f /t 0 (left to the default, like all other settings).
Does "path" refer to the command?  I've inserted the path to shutdown.exe to no avail.

**Comment by andy-portmen** [comment]
can you shutdown your Windows by running `shutdown /s /f /t 0` in cmd.exe?

> Does "path" refer to the command?

Yes

**Comment by ErikKrause** [comment]
Yes, `shutdown /s /f /t 0` (in a standard command window) shuts the computer down immediately.

**Comment by andy-portmen** [comment]
change the command to `cmd /k shutdown /s /f /t 500` and see if you still get the error or not.

You can cancel shutdown by `shutdown /a`

**Comment by ErikKrause** [comment]
Your extension works, it is the error message which is misleading if I press "Open" on the settings page. It says "If you still get this page even when the "Check Button" reports successful connection, it means the application cannot be accessed (path is not correct in the options page)" which seems to be plainly wrong.

Apart of that the wording is anything else than clear. Please use at least "path for the Command" instead of only "path". There is no "Check Button" there is a "Check Connection" button. And what is a successful connection? If it is the green popup then you should state exactly this. Also a positive remark would be much better than the stroke out lines top left (which, I assume, denote the successful installation steps).

Always remember that users don't know your extension like you do. They also don't usually know that webextensions requires this complicated installation, hence it would be a good idea to communicate this. 

But anyhow, many thanks for providing this as webextension just in time for FF 57!
